README.org

```diff
@@ -265,8 +265,24 @@
 **** Keymap-based replacement
      Using this method, which-key can display a custom string for a key
      definition in some keymap. There are two ways to define a keymap-based
-     replacement. The first is to use
-     =which-key-add-keymap-based-replacements=. The statement
+     replacement. The first is to define menu items,
+     [[https://www.gnu.org/software/emacs/manual/html_node/elisp/Simple-Menu-Items.html#Simple-Menu-Items][simple]]
+     or
+     [[https://www.gnu.org/software/emacs/manual/html_node/elisp/Extended-Menu-Items.html#Extended-Menu-Items][extended]],
```

Comment:
How about some examples of extended menu items for the readme?

Reply:
I added a small example of each in a recent commit, do those look sufficient?

Reply:
yes, thank you